dotCMS/src/integration-test/java/com/dotcms/saml/TestSamlConfigurationService.java

```diff
@@ -0,0 +1,33 @@
+package com.dotcms.saml;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.util.Config;
+import org.apache.felix.framework.OSGIUtil;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TestSamlConfigurationService {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5707914)